### PR TITLE
chore(privacy): detail into IP being hashed

### PR DESCRIPTION
**Pull request by PoolloverNathan**

### CODE REVIEW

**Review by nin0-dev — CHANGES_REQUESTED** [review]
nowhere have you said the IP was hashed

which makes this pr useless

**Comment by PoolloverNathan** [comment]
> nowhere have you said the IP was hashed
> 
> which makes this pr useless

the one thing I made the PR for I forgot to do

**Comment by PoolloverNathan** [comment]
> nowhere have you said the IP was hashed
> 
> which makes this pr useless

the one thing I made the PR for I forgot to do

**Comment by PoolloverNathan** [comment]
> > nowhere have you said the IP was hashed
> > 
> > which makes this pr useless
> 
> the one thing I made the PR for I forgot to do

@nin0-dev fixed

**Review by nin0-dev — APPROVED** [review]

File: privacy.html [hunk 1]
```diff
@@ -22,14 +22,13 @@
             We only use your data when strictly necessary; it is neither shared nor sold. Last
             updated on 10/20/2024.
```
Comment (Kodarru) [inline privacy.html:23]: 💀 I don't think today is the 20th

Comment (PoolloverNathan) [reply privacy.html:23]: guhhh

Comment (nin0-dev) [reply privacy.html:23]: guh

**Pull request merged**